charts/nr-k8s-otel-collector/templates/deployment-configmap.yaml

```diff
@@ -0,0 +1,334 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "newrelic.common.naming.fullname" . }}-deployment-config
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "newrelic.common.labels" . | nindent 4 }}
+data:
+  deployment-config.yaml: |
```

Comment:
same for the daemonset, the user should be able to customise this (not sure if we want it in the first iteration)

Reply:
`expose the config in values.yaml in followup pr`